### PR TITLE
Remove non-http references in documentation

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: lib/logstash/outputs/elasticsearch.rb [hunk 1]
```diff
@@ -16,7 +16,8 @@
 # This output only speaks the HTTP protocol. HTTP is the preferred protocol for interacting with Elasticsearch as of Logstash 2.0.
 # We strongly encourage the use of HTTP over the node protocol for a number of reasons. HTTP is only marginally slower,
 # yet far easier to administer and work with. When using the HTTP protocol one may upgrade Elasticsearch versions without having
-# to upgrade Logstash in lock-step. For those wishing to use the node or transport protocols please see the 'elasticsearch_java' gem.
+# to upgrade Logstash in lock-step. For those still wishing to use the node or transport protocols please see
+# the `logstash-output-elasticsearch_java` plugin.
```
Comment (talevy) [inline lib/logstash/outputs/elasticsearch.rb]: should we provide a link here?

File: lib/logstash/outputs/elasticsearch.rb [hunk 2]
```diff
@@ -101,23 +102,25 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
...
   config :port, :validate => :string, :default => 9200
 
-  # This plugin uses the bulk index api for improved indexing performance.
-  # To make efficient bulk api calls, we will buffer a certain number of
+  # This plugin uses the bulk index API for improved indexing performance.
+  # To make efficient bulk API calls, we will buffer a certain number of
   # events before flushing that out to Elasticsearch. This setting
   # controls how many events will be buffered before sending a batch
-  # of events.
+  # of events. Increasing the `flush_size` has an effect on Logstash's heapsize.
```
Comment (talevy) [inline lib/logstash/outputs/elasticsearch.rb]: s/heapsize/heap size?

File: lib/logstash/outputs/elasticsearch.rb [hunk 3]
```diff
@@ -101,23 +102,25 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
...
 
-  # This plugin uses the bulk index api for improved indexing performance.
-  # To make efficient bulk api calls, we will buffer a certain number of
+  # This plugin uses the bulk index API for improved indexing performance.
+  # To make efficient bulk API calls, we will buffer a certain number of
   # events before flushing that out to Elasticsearch. This setting
   # controls how many events will be buffered before sending a batch
-  # of events.
+  # of events. Increasing the `flush_size` has an effect on Logstash's heapsize. 
+  # Remember to also increase the heapsize if you are sending big documents or have increased the
```
Comment (talevy) [inline lib/logstash/outputs/elasticsearch.rb]: should we point to the documentation for this here? or at least mention `LS_HEAP_SIZE`?

**Comment by suyograo** [comment]
@talevy done

**Comment by talevy** [comment]
LGTM

**Comment by jordansissel** [comment]
LGTM

**Pull request closed without merging**